app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.players.player import Player
+
+
+def calculate_team_total_rating(team: list[Player]) -> int:
+    return sum(player.get_rating() for player in team)
+
+
+def elves_concert(elves: list) -> None:
```

Comment:
Indicate a list of what instances you expect to obtain in the type annotation in the future, not only for `team` variable